contributivity.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Sep 29 17:31:56 2019
+
+@author: RGOUSSAULT
+"""
+
+
+class Contributivity:
+  def __init__(self, name='', contributivity_scores=[], computation_time=0):
+    self.name = name
+    self.contributivity_scores = contributivity_scores
+    self.computation_time = computation_time
+    
+    
+  def __str__(self):
+    output = self.name + '\n'
+    output += 'computation time: ' + str(self.computation_time) + ' s' + '\n'
```

Comment:
Looks like the whitespace before the "s" of "seconds" is a typo - it would be prettier without.

Reply:
Hum no it's not a typo :)  --> https://english.stackexchange.com/questions/2794/punctuation-with-units